src/main/java/frc/robot/subsystems/ShooterSystem.java

```diff
@@ -0,0 +1,78 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotMap;
+
+public class ShooterSystem extends SubsystemBase {
+    private final CANSparkMax motorLT;
+    private final CANSparkMax motorRT;
+    private final CANSparkMax motorLB;
+    private final CANSparkMax motorRB;
+    private final RelativeEncoder encoderLT;
+    private final RelativeEncoder encoderRT;
+    private final RelativeEncoder encoderLB;
+    private final RelativeEncoder encoderRB;
+
+
+
+    public ShooterSystem(){
+        motorLT = new CANSparkMax(RobotMap.SHOOTER_MOTOR_LEFT_TOP, CANSparkLowLevel.MotorType.kBrushless);
+        motorRT = new CANSparkMax(RobotMap.SHOOTER_MOTOR_RIGHT_TOP, CANSparkLowLevel.MotorType.kBrushless);
+        motorLB = new CANSparkMax(RobotMap.SHOOTER_MOTOR_LEFT_BOTTOM, CANSparkLowLevel.MotorType.kBrushless);
+        motorRB = new CANSparkMax(RobotMap.SHOOTER_MOTOR_RIGHT_BOTTOM, CANSparkLowLevel.MotorType.kBrushless);
+
+        motorRT.restoreFactoryDefaults();
+        motorLT.restoreFactoryDefaults();
+        motorLB.restoreFactoryDefaults();
+        motorRB.restoreFactoryDefaults();
+
+        motorRT.setInverted(true);
+        motorRB.setInverted(true);
+
+        encoderLT = motorLT.getEncoder();
+        encoderLB = motorLB.getEncoder();
+        encoderRT = motorRT.getEncoder();
+        encoderRB = motorRB.getEncoder();
+    }
+    public void stop(){
+        motorLT.stopMotor();
+        motorLB.stopMotor();
+        motorRB.stopMotor();
+        motorRT.stopMotor();
+    }
+
+    public void rotate(){
+        motorLT.set(0.5);
+        motorRB.set(0.5);
+        motorLB.set(0.5);
+        motorRT.set(0.5);
+    }
+
+
+    public double getVelocityLB(){
+        return encoderLB.getVelocity();
+    }
+
+    public double getVelocityLT(){
+        return encoderLT.getVelocity();
+    }
+    public double getVelocityRT(){
+        return encoderRT.getVelocity();
+    }
+    public double getVelocityRB(){
+        return encoderRB.getVelocity();
+    }
+
+    @Override
+    public void periodic() {
+        SmartDashboard.putNumber("left top:", getVelocityLT());
+        SmartDashboard.putNumber("left bottom", getVelocityLB());
+        SmartDashboard.putNumber("right top", getVelocityRT());
+        SmartDashboard.putNumber("right bottom", getVelocityRB());
```

Comment:
make those changes for all four rows that refer to the SmartDashboard display:
1. no spaces and colon, that takes space on the display
2. more specific description that refer to the system. somthig like- ShooterRightBottomMotor

Reply:
done